### PR TITLE
Add example of ownership transfer

### PR DESCRIPTION
**Pull request by aknuds1**
Add an example program demonstrating ownership transfer.

### CODE REVIEW

File: examples/ownership-transfer/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Ownership Transfer Example
+This is an example of using the SDK to transfer ownership of a Tupelo chain tree from one
```
Comment (tobowers) [inline examples/ownership-transfer/README.md]: ChainTree

File: examples/ownership-transfer/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+entity to another. We are here imagining that two people, Alice and Bob, wish to
```
Comment (tobowers) [inline examples/ownership-transfer/README.md]: s/entity/owner

File: examples/ownership-transfer/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "ownership-transfer",
+  "version": "1.0.0",
+  "description": "An example of ownership transfer with Tupelo",
+  "main": "transfer.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (tobowers) [inline examples/ownership-transfer/package.json]: maybe a "run" here?

**Comment by aknuds1** [comment]
> looks generally good! Was going with JS over TS a deliberate decision?

Actually no! I didn't really think of TS since I'm not used to it, but I guess TS is the standard in this project? I can rewrite it to TS.

File: examples/ownership-transfer/package.json (hunk 3 above)
Comment (aknuds1) [reply examples/ownership-transfer/package.json]: @tobowers I don't think a bare `run` will work, I added `start` instead, which might be what you want?

**Comment by aknuds1** [comment]
@tobowers Please have a look at my conversion to TypeScript.

File: examples/ownership-transfer/src/@types/interface-datastore/index.d.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+declare module '*';
```
Comment (tobowers) [inline examples/ownership-transfer/src/@types/interface-datastore/index.d.ts]: I guess this is a little nicer... I'll copy this

File: examples/ownership-transfer/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
```
Comment (zonotope) [inline examples/ownership-transfer/src/index.ts]: Will the node interpreter automatically run typescript without any modification when a ts script with this shebang line is invoked?

Comment (aknuds1) [reply examples/ownership-transfer/src/index.ts]: Good catch @zonotope - leftover from before I rewrote to TS.

**Pull request merged**